code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -132,6 +132,10 @@
 /datum/chemical_reaction/reagent_explosion/tatp_explosion/proc/UpdateInfo()
 	required_temp = 550 + rand(-49,49)
 
+/datum/chemical_reaction/reagent_explosion/tatp_explosion/on_reaction(datum/reagents/holder, created_volume)
+	strengthdiv = created_volume == 2100 ? 1 : (created_volume*created_volume)/3000000 //formula that increases the potency of the explosion the more chem is used making tatp a top-tier instead of mid-tier
```

Comment:
shouldn't this check for >= instead of ==?

Reply:
also, isn't a higher strengthdiv worse than a lower strengthdiv?

Reply:
a 0.1u sample of tatp being able to be detonated in a 30000000 (not an exaggeration) power explosion is the definition of game balance

Reply:
for reference, a power explosion has a light explosion radius of 7745, a heavy explosion radius of 3872, and a devastating explosion radius of 1936 (all of those radii include the epicenter tile before moving outwards, but that doesn't really matter at this scale)

Reply:
I think what you're looking for is 3000000/(created_volume*created_volume)

Reply:
> a 0.1u sample of tatp being able to be detonated in a 30000000 (not an exaggeration) power explosion is the definition of game balance

oh shit i forgot the other part of the equasion. good eye mate

Reply:
i changed it to be linear i think it is fixed now